app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def index
+
+      channels = Channel.all.map { |channel| channel.id }
+      data = channels.to_json
+      render(status: 200, json: data)
+
+  end
+
+  def show
+
+      begin
```

Comment:
We don't have to have this begin here, we can just rescue down below like this

```ruby
def some_method
  danger_danger
  true # good response
rescue Error
  false # error response
end
```

Reply:
Added ✔️